client/src/components/subscription/subscriptionForm.js

```diff
@@ -124,103 +126,142 @@ class SubscriptionForm extends React.Component {
+          <label className="col d-flex form-label align-items-center col-md-6" htmlFor="subscription-nickname">
+            <div className="col-4 col-md-3">Nickname:</div>
+            <div className="col-8 col-md-7">
+              <input className="form-control" type="text" placeholder="e.g. My favorite streaming site"name="subscription-nickname" value={nickname} onChange={(event) => this.handleChange(event, 'nickname')}  /> 
+            </div>
           </label>
 
-          <label htmlFor="subscription-reminder-days">
-            Reminder Days: 
-            <input type="number" name="subscription-reminder-days" min="0" value={reminderDays} onChange={(event) => this.handleChange(event, 'reminderDays')} required />
+          <label className="col d-flex form-label align-items-center col-md-6" htmlFor="subscription-reminder-days">
+            <div className="col-4 col-md-3">Reminder Days:</div>
+            <div className="col-8 col-md-7">
+              <input className="form-control" type="number" name="subscription-reminder-days" min="0" value={reminderDays} onChange={(event) => this.handleChange(event, 'reminderDays')} required />
+            </div>
           </label>
 
-          <label htmlFor="due-date-select">
-            Repeat:
-            <select id="due-date-select" value={frequency} onChange={(event) => this.handleChange(event, 'frequency')} required >
-              <option value="" defaultValue>--Please Choose an Option--</option>
-              <option value="yearly">Yearly</option>
-              <option value="monthly">Monthly</option>
-              <option value="weekly">Weekly</option>
-              <option value="daily">Daily</option>
-            </select>  
+          <label className="col d-flex form-label align-items-center col-md-6" htmlFor="subscription-amount">
+            <div className="col-4 col-md-3">Amount:</div>
+            <div className="col-8 col-md-7">
+              <input className="form-control" type="number" name="subscription-amount" min="0" step="0.01" value={amount} onChange={(event) => this.handleChange(event, 'amount')} required />
+            </div>
           </label>
 
-          {this.renderSwitch(frequency)}
-          
-          <label htmlFor="subscription-amount">
-            Amount: 
-            <input type="number" name="subscription-amount" min="0" step="0.01" value={amount} onChange={(event) => this.handleChange(event, 'amount')} required />
+          <label className="col d-flex form-label align-items-center col-md-6" htmlFor="due-date-select">
+            <div className="col-4 col-md-3">Repeat:</div>
+            <div className="col-8 col-md-7">
+              <select className="form-control" id="due-date-select" value={frequency} onChange={(event) => this.handleChange(event, 'frequency')} required >
```

Comment:
The dropdown doesn't look like a select dropdown and instead looks like a text input field b/c it doesn't have the little arrow on the right side. I'd suggest using the `form-select` bootstrap class for this instead of the `form-control` that you're currently using

```suggestion
              <select className="form-select" id="due-date-select" value={frequency} onChange={(event) => this.handleChange(event, 'frequency')} required >
```